tests/integration/devfile/cmd_devfile_describe_test.go

```diff
@@ -107,4 +127,66 @@ var _ = Describe("odo devfile describe command tests", func() {
 		})
 	})
 
+	Context("devfile has missing metadata", func() {
+		// Reference: https://github.com/openshift/odo/issues/4815
```

Comment:
I don't think we need to keep referring an issue in the code unless it's something that needs to be fixed as a part of some TODO.

Reply:
I figured since this is an unusual scenario, something that we haven't taken into account before, it might help to reference the issue in order to avoid confusion.

Reply:
I agree with the unusual part. But I saw the issue getting referenced more than a few times. Besides, `git blame` would always bring someone to this PR and this PR can take them to issue 4815. So, IMO, it's OK to not mention it in the code unless it's a part of TODO.